x/params/store.go

```diff
@@ -0,0 +1,20 @@
+package params
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/x/params/store"
+)
+
+// re-export types from store
+type (
+	Store         = store.Store
+	ReadOnlyStore = store.ReadOnlyStore
+	ParamStruct   = store.ParamStruct
+	KeyFieldPairs = store.KeyFieldPairs
+)
+
+// UnmarshalParamsFromMap deserializes parameters from a given map. It returns
```

Comment:
is this being used anywhere?

Reply:
It was intended for querying but seems like we can use custom queriers instead, I will delete this.